Query for non-default eventbus eventbridge rules is querying default eventbus

**Describe the bug**
A clear and concise description of what the bug is.

Running `select * from aws_eventbridge_rule` will return all rules for all eventbuses. However, details such as `event_pattern` are null because the logs show the plugin is querying the `default` eventbus when it should be querying the non-default matching one for the rule.

Example log for rule on non-deault eventbus:
`[ERROR] 1676508541484: aws_eventbridge_rule.getAwsEventBridgeRule: api_error="operation error EventBridge: DescribeRule, https response error StatusCode: 400, RequestID: 3cae4cbe-6f88-493d-9c26-eb75a64252a8, ResourceNotFoundException: Rule EventRuleNameHere does not exist on EventBus default."`

**Steampipe version (`steampipe -v`)**
v0.18.6

**Plugin version (`steampipe plugin list`)**
v0.92.2

**To reproduce**
`select * from aws_eventbridge_rule`

**Expected behavior**
Correct `DescribeRule` for all rules

**Additional context**
Add any other context about the problem here.

--- a/aws/table_aws_eventbridge_rule.go
+++ b/aws/table_aws_eventbridge_rule.go
@@ -208,9 +208,11 @@ func listAwsEventBridgeRules(ctx context.Context, d *plugin.QueryData, h *plugin
 
 func getAwsEventBridgeRule(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
 
+	var eventbusname string
 	var name string
 	if h.Item != nil {
 		name = *h.Item.(*eventbridge.DescribeRuleOutput).Name
+		eventbusname = *h.Item.(*eventbridge.DescribeRuleOutput).EventBusName
 	} else {
 		name = d.EqualsQuals["name"].GetStringValue()
 	}
@@ -228,7 +230,8 @@ func getAwsEventBridgeRule(ctx context.Context, d *plugin.QueryData, h *plugin.H
 
 	// Build the params
 	params := &eventbridge.DescribeRuleInput{
-		Name: &name,
+		EventBusName: &eventbusname,
+		Name:         &name,
 	}
 
 	// Get call

--- a/aws/table_aws_eventbridge_rule.go
+++ b/aws/table_aws_eventbridge_rule.go
@@ -20,7 +20,7 @@ func tableAwsEventBridgeRule(_ context.Context) *plugin.Table {
 		Name:        "aws_eventbridge_rule",
 		Description: "AWS EventBridge Rule",
 		Get: &plugin.GetConfig{
-			KeyColumns: plugin.SingleColumn("name"),
+			KeyColumns: plugin.AllColumns([]string{"name", "event_bus_name"}),
 			IgnoreConfig: &plugin.IgnoreConfig{
 				ShouldIgnoreErrorFunc: shouldIgnoreErrors([]string{"ResourceNotFoundException", "ValidationException"}),
 			},
@@ -208,13 +208,14 @@ func listAwsEventBridgeRules(ctx context.Context, d *plugin.QueryData, h *plugin
 
 func getAwsEventBridgeRule(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
 
-	var eventbusname string
-	var name string
+	var name, eventbusname string
+
 	if h.Item != nil {
 		name = *h.Item.(*eventbridge.DescribeRuleOutput).Name
 		eventbusname = *h.Item.(*eventbridge.DescribeRuleOutput).EventBusName
 	} else {
 		name = d.EqualsQuals["name"].GetStringValue()
+		eventbusname = d.EqualsQuals["event_bus_name"].GetStringValue()
 	}
 
 	// Create Session
@@ -229,6 +230,7 @@ func getAwsEventBridgeRule(ctx context.Context, d *plugin.QueryData, h *plugin.H
 	}
 
 	// Build the params
+	// If you do not provide value for the EventBusName parameter, By Default DescribeRule API, will only look into the default event bus.
 	params := &eventbridge.DescribeRuleInput{
 		EventBusName: &eventbusname,
 		Name:         &name,

--- a/aws/table_aws_eventbridge_rule.go
+++ b/aws/table_aws_eventbridge_rule.go
@@ -208,14 +208,14 @@ func listAwsEventBridgeRules(ctx context.Context, d *plugin.QueryData, h *plugin
 
 func getAwsEventBridgeRule(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
 
-	var name, eventbusname string
+	var name, eventBusName string
 
 	if h.Item != nil {
 		name = *h.Item.(*eventbridge.DescribeRuleOutput).Name
-		eventbusname = *h.Item.(*eventbridge.DescribeRuleOutput).EventBusName
+		eventBusName = *h.Item.(*eventbridge.DescribeRuleOutput).EventBusName
 	} else {
 		name = d.EqualsQuals["name"].GetStringValue()
-		eventbusname = d.EqualsQuals["event_bus_name"].GetStringValue()
+		eventBusName = d.EqualsQuals["event_bus_name"].GetStringValue()
 	}
 
 	// Create Session
@@ -230,9 +230,9 @@ func getAwsEventBridgeRule(ctx context.Context, d *plugin.QueryData, h *plugin.H
 	}
 
 	// Build the params
-	// If you do not provide value for the EventBusName parameter, By Default DescribeRule API, will only look into the default event bus.
+	// Always provide an event bus name since the default event bus is used if not specified
 	params := &eventbridge.DescribeRuleInput{
-		EventBusName: &eventbusname,
+		EventBusName: &eventBusName,
 		Name:         &name,
 	}
 

